pkgs/applications/networking/browsers/webmacs/default.nix

```diff
@@ -0,0 +1,67 @@
+{ lib
+, fetchFromGitHub
+, python3Packages
+, qtbase
+, herbstluftwm
+}:
+
+python3Packages.buildPythonApplication rec {
+  pname = "webmacs";
+  version = "0.8";
+
+  src = fetchFromGitHub {
+    owner  = "parkouss";
+    repo   = "webmacs";
+    rev    = "d692ae8c2bf82cce854f34cb3d762932fe7fdc8b";
+    fetchSubmodules = true;
+    sha256 = "1hzb9341hybgrqcy1w20hshm6xaiby4wbjpjkigf4zq389407368";
+  };
+
+  propagatedBuildInputs = with python3Packages; [
+    python3Packages.pyqt5_with_qtwebkit
+    python3Packages.dateparser
+    python3Packages.jinja2
+  ];
+
+  buildInputs = [
+    python3Packages.pygments
```

Comment:
I think this should be in propagated build inputs but tbh I have not the faintest idea what the difference is

Reply:
My understanding was that `buildInputs` are only available at build time, while `propagatedBuildInputs` are also available at runtime, but that may be completely wrong. When I originally made the package, I took care to only stick things in propagated, if the browser didn't work without them ... but there may well have been holes in my ad-hoc testing.